TROUBLESHOOTING.md

```diff
@@ -44,3 +44,10 @@ on GNU Linux, it may be something like
 
 ### Grid-runner
 - When using grid-runner to run grid, individual services have to sometimes be restarted when switching branches
+
+### Accepting certificates
+- You have to manually accept the certificates for the urls for the different grid services.
```

Comment:
as this is a problem with the cert from dev-nginx, maybe this note should be there instead?